data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1233,11 +1233,16 @@ fleet "Elenchus Tribute"
 		"Vanguard (Missile)"
 		"Firebird"
 
-event "Elenchus can be dominated"
+# They suspect you're coming so they're on alert, and non-essential ships aren't flying about.
+# Also, removing the warship spawn stops your NPC escorts from getting overwhelmed inappropriately.
+event "elenchus can be dominated"
 	planet "Elenchus"
 		tribute 5000
 			threshold 2400
 			fleet "Elenchus Tribute"
+	system "Devil-Hide"
+		remove fleet "Devil-Run Warships"
+		remove fleet "Devil-Run Miners"
```

Comment:
just doing this by itself causes a huge plothole if you ask me
like, why dont they help with the tribute fleet? earlier dialog by Choot'k mentioned they'd cover you from incoming ships, too, I think?
I'd move this to another PR where we would replace it by smth else